electron/main.js

```diff
@@ -69,7 +69,7 @@ app.on("ready", () => {
   server.setupHTTPServer(port);
 
   // setup Database
-  database.openConnection(false);
+  database.openConnection(true);
```

Comment:
would be nice to put this into a variable so it's clear what it means, like `const isInMemory = false` (something like that)

Reply:
I added a variable.